lighthouse-cli/performance-experiment/report/injected-script.js

```diff
@@ -30,17 +30,39 @@ window.addEventListener('DOMContentLoaded', _ => {
   rerunButton.style.display = 'inline-block';
 
   rerunButton.addEventListener('click', () => {
-    rerunButton.setAttribute('status', 'running');
-    rerunLighthouse().then(() => {
+    rerunButton.setAttribute('data-status', 'running');
```

Comment:
Not a fan of `data-*` since you're not really storing data. These represent state, so I think its fine to have it without the `data-*` prefix. If its simply for styling, just use a class instead.

`rerunButton.classList.add('...')`